docs/studio/Dashboards.md

```diff
@@ -0,0 +1,60 @@
+# Dashboards
+
+## What is a dashboard?
+
+A `Dashboard` is simply a query with a description and label. This will be used by the `Workspace` to query against the view paired with it inside the `Workspace` configuration. This way, you can write one query, and re-use it against multiple potential views.
+
+An example `Dashbaord` resource looks like this:
+
+```json
+{
+  "@context": "https://bluebrainnexus.io/studio/context",
+  "@type": "StudioDashboard",
+  "dataQuery": "# This is a simple example query\n# You can directly edit this\nprefix nxv: <https://bluebrain.github.io/nexus/vocabulary/>\nprefix s: <http://schema.org/>\nSELECT DISTINCT ?self ?familyName ?givenName\nWHERE {\n?s nxv:constrainedBy <https://neuroshapes.org/dash/person> ;\n  nxv:self ?self ;\n  s:familyName ?familyName ; \n  s:givenName ?givenName\n}\nLIMIT 20",
```

Comment:
Do we need to keep here `# This is a simple example query\n# You can directly edit this\nprefix nxv...` ?